buildpack.toml

```
@@ -15,39 +15,48 @@
 api = "0.2"
 
 [buildpack]
-id = "io.projectriff.command"
-name = "Command Function Buildpack"
-version = "0.2.0-BUILD-SNAPSHOT"
+id       = "riff-buildpacks/command-function"
```

not sure about this suggestion, take it or leave it

```suggestion
id       = "projectriff/command-function"
```